package.json

```diff
@@ -90,8 +93,21 @@
       "browser",
       "assert",
       "$",
-      "XMLHttpRequest"
-    ]
+      "XMLHttpRequest",
+      "JSDOM",
```

Comment:
why are there so many global functions? e.g anything with "getBackLink" i'd imagine shouldn't be a global at all